### PR TITLE
Data base test blake

### PR DESCRIPTION
**Pull request by bserge13**
## Description

Removal of joins table and controller. Removal of headers in dogs_request for DogsFacade testing on FE

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] all tests pass

### CODE REVIEW

**Review by EricBelongea — CHANGES_REQUESTED** [review]
We use the joins table.

File: spec/requests/api/v1/dogs_request_spec.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@
                     })
       headers = {"CONTENT_TYPE" => "application/json"}
     
-      post api_v1_user_dogs_path(user.id, dog_params), headers: headers, params: JSON.generate(dog: dog_params)
+      post api_v1_user_dogs_path(user.id, dog_params), params: (dog_params)
```
Comment (EricBelongea) [inline spec/requests/api/v1/dogs_request_spec.rb:88]: Good change here

**Review by ttakahashi1591 — APPROVED** [review]
Thanks for getting this set up!

File: db/migrate/20240110042434_drop_user_socials.rb
Comment (EricBelongea) [inline db/migrate/20240110042434_drop_user_socials.rb:1]: I think you're right, but this still scares me haha

**Review by EricBelongea — APPROVED** [review]
I think I approve this. Scary to drop the table this close to due-date, but I think you're right and that we aren't using the user_scoials table anymore

**Pull request merged**